### PR TITLE
[rbs] Add downloader for new provider

### PR DESCRIPTION
**Pull request by n1chre**
Implemented fetching and converting to NVD for risk based security

### CODE REVIEW

File: providers/lib/download/http.go [hunk 1]
```diff
@@ -38,21 +38,25 @@ func (e Err) Error() string {
 	return fmt.Sprintf("http error %s:\n %q", e.Status, e.Body)
 }
 
-// Query will query the given URL and then return the response if response status is HTTP.OK
-// if status is not HTTP.OK, returns Err
+// Get will call GetWithClient using the default client
 func Get(url string, header http.Header) (*http.Response, error) {
+	if client, err := Client(); err != nil {
+		return nil, fmt.Errorf("can't obtain default client: %v", err)
+	} else {
```
Comment (skogtwin) [inline providers/lib/download/http.go]: I'd dropped `else` -- the clauses are not at the same level, the one above handles the error, the one below is the expected flow of the program.

File: providers/rbs/api/client.go [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+	c := Client{
+		client:    conf.Client(ctx),
+		baseURL:   baseURL,
+		userAgent: userAgent,
+	}
+
+	return &c, nil
+}
+
+func (c *Client) FetchAllVulnerabilitiesAfterVulndbID(vulndbID int) (<-chan *schema.Vulnerability, error) {
```
Comment (skogtwin) [inline providers/rbs/api/client.go]: super-nitpick, but i don't like all these [Ff]etchAllVulnerabilities.* names. I'd dropped the 'Vulnerabilities' part.

File: providers/rbs/converter/converter.go [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+		if err != nil {
+			log.Println(err)
+			continue
+		}
+		feed.CVEItems = append(feed.CVEItems, converted)
+	}
+	return &feed
+}
+
+func convert(item *srcSchema.Vulnerability) (*dstSchema.NVDCVEFeedJSON10DefCVEItem, error) {
```
Comment (skogtwin) [inline providers/rbs/converter/converter.go]: I'd make it a method of `srcSchema.Vulnerability` and ditched the `Convert` above -- it feels like something that whaterver sets up the input channel should do.

File: providers/rbs/api/client.go (hunk 2 above)
Comment (n1chre) [reply providers/rbs/api/client.go]: Will make it uniform in my next diffs

File: providers/rbs/converter/converter.go (hunk 3 above)
Comment (n1chre) [reply providers/rbs/converter/converter.go]: Yeah, it's going to be like that in the next diffs, unifying all converters. For now, it does the same thing as others do

**Pull request merged**